core/README.md

```diff
@@ -0,0 +1,146 @@
+# Vega core
+
+A decentralised trading platform that allows pseudo-anonymous trading of derivatives on a blockchain.
+
+**Vega** provides the following core features:
+
+- Join a Vega network as a validator or non-consensus node.
+- [Governance](./governance/README.md) - proposing and voting for new markets
+- A [matching engine](./matching/README.md)
+- [Configure a node](#configuration) (and its [APIs](#apis))
+- Manage authentication with a network
+- [Run scenario tests](./integration/README.md)
+- Support settlement in cryptocurrency (coming soon)
+## Links
+
+- For **new developers**, see [Getting Started](GETTING_STARTED.md).
+- For **updates**, see the [Change log](CHANGELOG.md) for major updates.
+- For **architecture**, please read the [documentation](docs/index.md) to learn about the design for the system and its architecture.
+- Please [open an issue](https://github.com/vegaprotocol/vega/issues/new) if anything is missing or unclear in this documentation.
+
+<details>
+  <summary><strong>Table of Contents</strong> (click to expand)</summary>
+
+<!-- toc -->
+
+- [Vega](#vega)
+  - [Links](#links)
+  - [Installation](#installation)
+  - [Configuration](#configuration)
+    - [Files location](#files-location)
+  - [Vega node wallets](#vega-node-wallets)
+    - [Using Ethereum Clef wallet](#using-ethereum-clef-wallet)
+      - [Automatic approvals](#automatic-approvals)
+      - [Importing and generation account](#importing-and-generation-account)
+  - [API](#api)
```

Comment:
should this be:
`../datanode#data-node` readme content or `./api` directory?

Reply:
nope this is fine.